melosys-eessi-app/src/main/java/no/nav/melosys/eessi/integration/pdl/dto/PDLIdent.java

```diff
@@ -14,4 +14,12 @@
     public boolean erFolkeregisterIdent() {
         return "FOLKEREGISTERIDENT".equals(gruppe);
     }
+
+    public boolean erAktørID() {
+        return "AKTORID".equals(gruppe);
+    }
+
+    public boolean erNPID() { //NAV PersonIdent
+        return "NPID".equals(gruppe);
```

Comment:
Jeg er nysgjerrig på om vi kommer til å bruke NPID i Melosys.

Reply:
Ja spørs om vi egentlig trenger å forholde oss til det i hvert fall. Men ved en eventuell automatisk d-nummer rekvirering, så er NPID høyaktuell 😄  Men det ligger nok litt frem i tid

Reply:
Fortsatt nysgjerrig 😄 Hvordan kunne meessi avgjøre om det skal brules NPID eller DNR?

Reply:
Alt i alt så spiller det egentlig ingen rolle, da PDL ikke bryr seg om hva vi sender inn. Så er nok helt unødvendig av meg å referere til den koden her 😄 Fjerner den

Er bare noe vi så vidt har touchet innom da vi har nevnt automatisk rekvirering. At vi alltid kunne ha rekvirert opprettet en NPID for personer NAV ikke har noen ident på. Da slipper Norge å gå tom for d-nummer om vi skulle ha rekvirert for alle lastebilsjåfører som så vidt kjører gjennom Norge 😄 

Oppsummert så tviler jeg vi noen gang i nær fremtid trenger å referere spesifikt til den ident-typen